volmdlr/wires.py

```diff
@@ -37,13 +37,12 @@ def argmax(list_of_float):
     Returns the max value and the argmax.
     """
     pos_max, max_float = 0, list_of_float[0]
-    for pos, fl in enumerate(list_of_float):
+    for pos, float_ in enumerate(list_of_float):
         if pos == 0:
             continue
-        else:
-            if fl > max_float:
-                max_float = fl
-                pos_max = pos
+        if float_ > max_float:
+            max_float = float_
+            pos_max = pos
     return max_float, pos_max
 
 
```

Comment:
`def argmin(self, list_of_numbers):`

Reply:
maybe we should move theses function into a utils module

Reply:
Yes, it would be a good idea. Maybe in a another PR towards dev?